indra/newview/scripts/lua/LLChatListener.lua

```diff
@@ -0,0 +1,41 @@
+local fiber = require 'fiber'
+local inspect = require 'inspect'
+
+local LLChatListener = {}
+local waitfor = {}
+
+function LLChatListener:new()
+    local obj = setmetatable({}, self)
+    self.__index = self
+    obj.name = 'Chat_listener'
+
+    return obj
+end
+
+function LLChatListener:handleMessages(event_data)
+    --print(inspect(event_data))
```

Comment:
Since the anticipated usage is to override `handleMessages()`, maybe leave this uncommented? If the coder forgets to do that, at least they'll get a visual reminder.

Oh, I see: your example override method forwards the call to this base-class method. Okay.

Reply:
Uncommented and updated example script too.